src/applications/claims-status/tests/e2e/fixtures/mocks/lighthouse/claim-detail-open.json

```diff
@@ -166,7 +166,7 @@
           "receivedDate": "2022-01-04",
           "requestedDate": null,
           "suspenseDate": null,
-          "id": 4,
```

Comment:
Updated the ids of the tracked items here so that they were sequential and easier to follow